code-organization/README.md

```diff
@@ -0,0 +1,12 @@
+# Packwerk
```

Comment:
I like that you title the page `code-organization`, but would carry that to this title, then have a "Packwerk" section.

The reason I really like this is because it sparked an idea of documenting how we use "Form Models" or "Service Models" (or maybe they are separate things?) to organize our code over using "Service Objects".

Reply:
I like where you are going @craineum  and think we possibly need more of that documented. 

We do have a section already with a few OO things here https://github.com/BuoySoftware/guides/tree/1cd099841ca46aeae3aa25d79b9d0154c762db3a/object-oriented-design

So I actually wonder if the OOD pages bullets should be moved here, or vice versa, if we should be putting a Packwerk section in the OOD page and later adding more around what else you have mentioned?